server/command_hooks.go

```diff
@@ -393,6 +421,207 @@ func (p *Plugin) executeCommandDialog(args *model.CommandArgs) *model.CommandRes
 	return &model.CommandResponse{}
 }
 
+func getDialogBasic() model.Dialog {
```

Comment:
Should these go into a seprate files as suggested via https://github.com/mattermost/mattermost-plugin-demo/blob/f3c20177a4d3263cfadfa48001ecbc1ee74a9121/server/dialog_samples.go?